File: Golang/beecrowd-challenges/bee-1015.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x float64
	fmt.Scan(&x)
	var y float64
	fmt.Scan(&y)
	var x2 float64
	fmt.Scan(&x2)
	var y2 float64
	fmt.Scan(&y2)

	calc := math.Sqrt(math.Pow(x2-x, 2) + math.Pow(y2-y, 2))

	fmt.Printf("%.4f", calc)
}

File: javascript/JavaScriptHackerRank/beecrowd/bee-1000-1037/bee-1015.js
const input = require('fs').readFileSync('./dev/stdin', 'utf8');
const lines = input.split('\n');
let lines1 = lines.shift().split(' ')
let lines2 = lines.shift().split(' ')

let x1 = Number(lines1.shift())
let x2 = Number(lines2.shift())
let y1 = Number(lines1.shift())
let y2 = Number(lines2.shift())

let result = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2))

console.log (result.toFixed(4))
